config/tests/test_urls.py

```diff
@@ -0,0 +1,7 @@
+from socialhome.tests.utils import SocialhomeTestCase
```

Comment:
I know this is the logical place for the test, but I feel still this file should be located in the root Socialhome app. Our project layout is maybe not optimal in this sense. So, I'd move this file to `socialhome/tests/test_api_docs.py`. It doesn't quite match the pattern, I see you've followed that closely, but I think the `config` module is maybe the wrong place for tests and code. Actually, due to this line in `pytest.ini`, this test is not even run by the CI: `testpaths = socialhome`